test/scripts/test-desktop.js

```diff
@@ -25,7 +25,7 @@ cp(path.join(root, 'fixtures'), path.join(TMPDIR, 'ssc-socket-test-fixtures'), {
 })
 
 try {
-  exec(`ssc build -r --test=./index.js ${!DEBUG ? '-o --headless --prod' : ''}`, {
+  exec(`ssc build -r --test=./index.js ${!DEBUG ? '-o  --prod' : ''}`, {
```

Comment:
why drop `--headless`?

Reply:
its already in `socket.ini` - if we want to debug without headless on desktop, we need to comment it out in the `socket.ini` file and then remove it here